src/Symfony/Component/Validator/Constraints/IbanValidator.php

```diff
@@ -163,9 +163,6 @@ class IbanValidator extends ConstraintValidator
         'YT' => 'FR\d{2}\d{5}\d{5}[\dA-Z]{11}\d{2}', // France
     ];
 
-    /**
-     * {@inheritdoc}
-     */
```

Comment:
I suppose that this comes from fabbot. This should be reverted as not related to this PR.